doku.php

```diff
@@ -11,6 +11,16 @@
 // update message version - always use a string to avoid localized floats!
 $updateVersion = "50";
 
+// let people know if they are running an unsupported version of PHP
+$required_php = "5.6";
+
+if (version_compare(phpversion(), $required_php, '<')) {
+
+  die('<h2 style="margin:1.2em;text-align:center;color:red;line-height:1.6;" >Error: You are currently running PHP/'.phpversion().'.<br>
+       <span style="color:#606060;" >DokuWiki requires PHP/'.$required_php.' or higher.</span></h3>');
```

Comment:
I wonder if this could use `nice_die()` (defined in `inc/init.php`). For this, probably `nice_die()` would need to be moved out of `inc/init.php` to ensure that no code that is incompatible with old PHP versions is executed.

Reply:
[Sorry this reply is only a test-input...]

Reply:
So far as I can see (but I am really a raw recruit in PHP programming and in DokuWiki core anymore) there is no significant benefit that make it warrantable to move nice_die() to doku.php